plugins/orchestrator/localregistry/nbinitfileregistry.go

```diff
@@ -0,0 +1,286 @@
+// Copyright (c) 2020 Pantheon.tech
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package localregistry
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	yaml2 "github.com/ghodss/yaml"
+	"github.com/go-errors/errors"
+	"github.com/golang/protobuf/proto"
+	"go.ligato.io/cn-infra/v2/config"
+	"go.ligato.io/cn-infra/v2/datasync"
+	"go.ligato.io/cn-infra/v2/datasync/kvdbsync/local"
+	"go.ligato.io/cn-infra/v2/datasync/resync"
+	"go.ligato.io/cn-infra/v2/datasync/syncbase"
+	"go.ligato.io/cn-infra/v2/db/keyval"
+	"go.ligato.io/cn-infra/v2/infra"
+	"go.ligato.io/vpp-agent/v3/client"
+	"google.golang.org/protobuf/encoding/protojson"
+	protoV2 "google.golang.org/protobuf/proto"
+)
+
+const (
+	registryName          = "nb-init-file-registry"
+	defaultNBInitFilePath = "nb-initial-config.yaml"
```

Comment:
I think we should lose the `nb-` prefix. There is no other initial config than NB - northbound one so does not add extra info for user, just makes the filename longer and can potentially confuse users.

Reply:
Ok, that sound good. It also that means to change the name of the registry, its config name and content (i mean config for the registry not the init config). Otherwise, that will confuse anyone looking into the code.

Reply:
Yes, alright

Reply:
Fixed in [`d7d34d3` (#1769)](https://github.com/ligato/vpp-agent/pull/1769/commits/d7d34d35dce453592232e386f79f677bdd5bfd86)